src/Symfony/Component/Console/Helper/Helper.php

```diff
@@ -59,6 +70,27 @@ public static function strlen(?string $string)
         return mb_strwidth($string, $encoding);
     }
 
+    /**
+     * Returns the length of a string, using mb_strlen if it is available.
+     * The length is related to how many bytes the string will use.
+     *
+     * @internal in 5.2
```

Comment:
since Symfony

Reply:
Most of the time we just do `@internal`. 

I've updated these two methods and their annotations say `@internal in Symfony 5.2` because they are soon to be non-internal.